business/namespaces.go

```diff
@@ -68,6 +68,15 @@ func NewNamespaceService(userClients map[string]kubernetes.ClientInterface, kial
 	}
 }
 
+// Returns a list of the given namespaces / projects
```

Comment:
GetClusterList returns a list of the clusters (actually the userClients for the NamespaceService)

Reply:
Updated